build/ci/templates/nuget-cache.yml

```diff
@@ -6,10 +6,10 @@ steps:
   - powershell: |
       New-Item -ItemType Directory -Force "${{ parameters.nugetPackages }}"
     displayName: Create nuget packages cache folder
-
+  
   - script: |
-      compact /c "/s:${{ parameters.nugetPackages }}"
-
+      compact /c "${{ parameters.nugetPackages }}"
+  
     condition: and(succeeded(), eq( variables['Agent.OS'], 'Windows_NT' ))
     displayName: Compress nuget package cache folder
```

Comment:
How is the cache supposed to work? The `Cache` task below has a condition on `enable_package_cache` which is never set to true.